src/js/views/eva-variant-view.js

```diff
@@ -730,42 +761,50 @@ EvaVariantView.prototype = {
         }
     },
 
-    _renderSummaryData: function (data) {
-        var _this = this;
-        var speciesName, organism;
+    _getSpeciesOrganismValues: function () {
         if (!_.isEmpty(this.speciesList)) {
             speciesName = _.findWhere(this.speciesList, {taxonomyCode: this.species.split("_")[0]}).taxonomyEvaName;
             organism = speciesName.substr(0, 1).toUpperCase() + speciesName.substr(1);
         } else {
             speciesName = this.species;
+            organism = speciesName;
         }
 
-        var getSummaryTableHeaderRow = function(summaryData) {
-            var header = '';
-            _.each(_.keys(summaryData), function(key) {
-                if (key === summaryDisplayFields.allelesMatch) {
-                    header += '<th><span title="' + allelesMatchToolTip + '">' + key + '</span></th>';
-                } else {
-                    header += '<th>' + key + '</th>';
-                }
-            });
-            return '<thead><tr>' + header + '</tr></thead>';
-        };
-        var getSummaryTableContentRow = function(summaryData) {
-            var rowContent = '';
-            _.each(_.keys(summaryData), function(key) {
-                var content = summaryData[key] ? summaryData[key]: '';
-                rowContent += '<td>' + content + '</td>';
-            });
-            return '<tr>' + rowContent + '</tr>';
-        };
+        return [speciesName, organism];
+    },
+
+    _getSummaryTableHeaderRow: function(summaryDisplayFields, summaryData) {
+        var allelesMatchToolTip = "1) Reference allele appears in the list of alleles that were submitted and 2) Locus orientation was determined definitively";
+        var header = '';
+        _.each(_.keys(summaryData), function(key) {
+            if (key === summaryDisplayFields.allelesMatch) {
+                header += '<th><span title="' + allelesMatchToolTip + '">' + key + '</span></th>';
+            } else {
+                header += '<th>' + key + '</th>';
+            }
+        });
+        return '<thead><tr>' + header + '</tr></thead>';
+    },
+
+    _getSummaryTableContentRow: function(summaryData) {
+        var rowContent = '';
+        _.each(_.keys(summaryData), function(key) {
+            var content = summaryData[key] ? summaryData[key]: '';
+            rowContent += '<td>' + content + '</td>';
+        });
+        return '<tr>' + rowContent + '</tr>';
+    },
+
+    _renderSummaryData: function (data) {
+        var _this = this;
+        var array = this._getSpeciesOrganismValues(); speciesName = array[0]; organism = array[1];
```

Comment:
why putting several statements in the same line? doesn't it hide those variables?